cni/pkg/cmd/root.go

```diff
@@ -112,6 +112,11 @@ var rootCmd = &cobra.Command{
 			if err != nil {
 				return fmt.Errorf("failed to instantiate ambient enablement selector: %v", err)
 			}
+
+			if cfg.InstallConfig.NativeNftables && cfg.InstallConfig.ForceIptablesBinary != "" {
+				log.Warn("NativeNftables is enabled along with ForceIptablesBinary. Using native nftables and ignoring iptables backend")
```

Comment:
I think we want to settle on 1 term for this and stick with it everywhere. The PR sort of alternates between "binary" and "backend". Personally, I think I slightly prefer "backend" but if folks like "binary" more I'm ok standardizing on that and stripping "backend" from anything a user would see.

Reply:
fixed in https://github.com/istio/istio/pull/57976/commits/ac00ba950c64e837f148309148183b5a03a376e1